spec/index.html

```diff
@@ -262,7 +262,7 @@ <h2>Introduction</h2>
       The terms YAML, "YAML document", "YAML representation graph",
```

Comment:
"YAML document" has been carefully edited out of most of this document, so perhaps it should be removed here as well? 
```suggestion
      The terms YAML, "YAML representation graph",
```

Reply:
The original text is ok to me. Let me just rebase the PR @gkellogg :)

Reply:
PR is merged, thanks.